### PR TITLE
Update Installation issue #81

### PR DESCRIPTION
**Pull request by me-pic**

### CODE REVIEW

**Comment by pbellec** [comment]
LGTM. @FrancoisPgm check it out and merge if you think this is ready. Fix #97

File: content/en/modules/installation/index.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ It is a good idea to associate this with your university e-mail (if you have one
 ### Discord
 
 Go to https://discord.com/download and download and install Discord.
-You will be invited to the BrainHack School Slack server via the e-mail with which you registered for the course.
+You will be invited to the BrainHack School Slack server via the e-mail with which you registered for the course. If you have not received the invitation, please reach out to one of the instructors.
```
Comment (FrancoisPgm) [inline content/en/modules/installation/index.md:71]: It is not the Slack server but Discord server.

**Review by FrancoisPgm — CHANGES_REQUESTED** [review]
Apart for the mention of Slack server instead of Discord server (that was already wrong last year), it looks perfect to me.

File: content/en/modules/installation/index.md (hunk 1 above)
Comment (me-pic) [reply content/en/modules/installation/index.md]: Info updated according to the reviews !

**Pull request merged**